### PR TITLE
7pm-4 nw - Adds `/api/slackAdmins` endpoint that returns a list of `SlackUser`s who are Slack admins

### PR DESCRIPTION
**Pull request by nicomwong**
New endpoint `/api/slackAdmins`that returns a list of `SlackUser`s who are Slack admins.

### CODE REVIEW

**Review by pconrad — DISMISSED** [review]
LGTM, pending test on qa site

**Review by karankhangura — APPROVED** [review]
LGTM!

**Comment by andrewhlu** [comment]
I noticed this PR is still marked as a "draft PR". @nicomwong, is this PR ready to be reviewed? If it is, please click the "Ready for review" button to take it out of draft mode.

File: src/main/java/edu/ucsb/mapache/repositories/SlackUserRepository.java [hunk 1]
```diff
@@ -12,4 +12,7 @@
 public interface SlackUserRepository extends MongoRepository<SlackUser, ObjectId> {
     @Query("{ 'profile.email': ?0}")
     List<SlackUser> findByEmail(String email);
+
+    @Query("{ 'is_admin': true")
```
Comment (andrewhlu) [inline src/main/java/edu/ucsb/mapache/repositories/SlackUserRepository.java:16]: This search query doesn't have a matching closing brace. Can we fix that?

**Review by andrewhlu — CHANGES_REQUESTED** [review]
One minor fix

File: src/main/java/edu/ucsb/mapache/repositories/SlackUserRepository.java (hunk 1 above)
Comment (nicomwong) [reply src/main/java/edu/ucsb/mapache/repositories/SlackUserRepository.java]: Done

**Review by karankhangura — APPROVED** [review]
LGTM

**Review by mara-downing — APPROVED** [review]
LGTM

**Pull request merged**